app/views/cohorts/_breadcrumbs.haml

```diff
@@ -1,3 +1,7 @@
 = content_for :crumbs do
   = link_to cohorts_path do
-    &laquo; Cohorts
\ No newline at end of file
+    &laquo; Cohorts
+  - if @cohort.present?
+    = link_to cohort_path(@cohort) do
+      &laquo;
+      = @cohort.name
```

Comment:
```suggestion
      Return to  #{@cohort.name}
```

Reply:
That would result in "Cohorts << Return to My Cohort" -- I'm not sure whether "Cohorts << My Cohort" which is what I implemented or "Return to My Cohort" is better.